draft-ietf-quic-multipath.md

```diff
@@ -381,16 +378,20 @@ with the same, unused Path ID. When the peer receives the PATH_CHALLENGE,
 it MUST pick a Connection ID with the same Path ID for sending the PATH_RESPONSE.
 
 When the multipath extension is negotiated, a client that wants to use an
-additional path MUST first initiate the Address Validation procedure
+additional path SHOULD first initiate the Address Validation procedure
 with PATH_CHALLENGE and PATH_RESPONSE frames as described in
-{{Section 8.2 of QUIC-TRANSPORT}}, unless it has previously validated
+{{Section 8.2 of QUIC-TRANSPORT}}. It MAY skip the validation and
+simply send packets on the paths if it has previously validated
 that address.
```

Comment:
We discussed in the meeting that we want a MUST here but don't say that path validation would require sending a path challenge but rather just refer to RFC9000 instead.

Reply:
I know we discussed that. But we also agreed that packets received from the peer should be processed and acknowledged. To acknowledge, the receiving peer has to create a "path context". We need to thread the needle through a fine hole...

The critical issue is that the peer address could be spoofed, as stated in section 9.2 of RFC 9000. Thus, address validation is required. It is not perfect.

I will try to rewrite that without any MUST/SHOULD/MAY, with simply a reference to RFC 9000.

Reply:
Yes, address validation is required, however, if you have valid the address previously or in a different way, you don't need to the the QUIC path challenge/response mechanism. Therefore the agreement in the session was to have a MUST for address validation but not explicitly say that it has to be performed by using path challenge/response instead but just provide a point to RFC9000. See my proposed wording.

Reply:
@mirjak, I think the commit [Rewrite per review feedback](https://github.com/quicwg/multipath/pull/442/commits/9f74f86bb43f5509f4f49e60e61aac40a1f6679a) incorporates the text that we agreed upon. Please review!